extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -40,17 +40,38 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 		return allFiles;
 	}
 
-	async openNotebook(resource: vscode.Uri): Promise<void> {
+	async openNotebook(resource: string): Promise<void> {
 		try {
 			let doc = await vscode.workspace.openTextDocument(resource);
```

Comment:
This is fine with it changing to a string? Why is this changing anyways - was this fixing a bug with the implementation earlier?

Reply:
Yes, I realized that vscode.workspace.openTextDocument() accepts a string as a parameter and not Uri